### DISCUSSION TITLE
Cell input should appear in notebook output area

### DISCUSSION THREAD

**Discussion by minsa110** [microsoft/vscode-jupyter#9961] [New Ideas]
Feedback from PyCon 2022

"The input box (from the Python's input() function) normally shows up on the notebook output space in classic Jupyter, but the input box is showing up on the top by the command palette. Is there a way for me to change it?"

### In Jupyter
![image](https://user-images.githubusercontent.com/10833267/167353829-53801ac1-d54e-4096-b605-8e2848c27b95.png)
After entering name:
![image](https://user-images.githubusercontent.com/10833267/167353837-6ba761e9-ffa8-4181-b2b8-64d20352fe11.png)

### In VS Code
![image](https://user-images.githubusercontent.com/10833267/167353035-fa67e4ff-c6c9-4b3c-b4c5-d58fedc1781c.png)
After entering name:
![image](https://user-images.githubusercontent.com/10833267/167353746-20244d8d-dc4c-4c7d-b386-1b5d843f393a.png)

**Comment by ArielG100** [comment]
I have the same problem. To present results of our code by exporting the file in pdf or html, **the results must show the input values** ​​that cause a specific output. Of course we can write extra code to display this input data, but that would be overwriting code.

**Comment by arthurkf** [comment]
I also missed this feature today. Input and output are far from each other, making understanding difficult. You have to write code to print input data to bypass.

**Comment by DonJayamanne** [comment]
Duplicate of https://github.com/microsoft/vscode-jupyter/issues/11732
